python/src/aac/context/lexeme.py

```diff
@@ -20,6 +20,19 @@ class Lexeme:
     source: str = attrib(validator=validators.instance_of(str))
     value: str = attrib(validator=validators.instance_of(str))
 
+    def __init__(self, location: SourceLocation, source: str, value: str):
```

Comment:
So, was similar behavior to what was being seen with `LanguageError` found to be happening with the `Lexeme` class then?

Reply:
I didn't see anywhere that had that issue, but it's never a bad idea to have an __init__ method anyway, and it could avoid a problem down the line.